src/app/contextBroker/contextBroker.cpp

```diff
@@ -406,6 +409,10 @@ PaArgument paArgs[] =
 
   { "-insecureNotif", &insecureNotif, "INSECURE_NOTIF", PaBool, PaOpt, false, false, true, INSECURE_NOTIF },
 
+#ifdef PARANOID_JSON_INDENT
+  { "-paranoidV1Indent", &paranoidV1Indent, "PARANOID_V1_INDENT", PaBool, PaHid, false, false, true, "you shouldn't use this ;)" },
```

Comment:
same same, nothing paranoid with giving a human readable format.
That actually might be a good name for it: "Human Readable Format":

So, I'd propose:
```
#ifdef HUMAN_READABLE_FORMAT ...
```

Reply:
It is not about being human readable... it is a about to be prepare in the rare case some NGSIv1 client breaks, so the "paranoid" about that. However, as I said in a previous comment, the name of the define is not really important.

Reply:
Ok

NTC